### ISSUE TITLE
Anonymous Objects

### ISSUE THREAD

**Issue by grant-d** [Microsoft/bond#50]
In Json.Net it is very simple to deserialize a Json stream into an anonymous object.
Is there a simple (native) way to do this in Bond, or do I need to roll my own?

Thanks

**Comment by sapek** [comment]
There isn't currently although this is a feature we are considering.

**Comment by eharshosh** [comment]
Any news in the regard?

**Comment by chwarr** [comment]
Nope, no one has looked into this in more detail. Is this something you'd be willing to help us design and implement?
